lib/pharos/ssh_command.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Pharos
+  class SSHCommand < UpCommand
+    usage "[OPTIONS] -- [COMMANDS] ..."
+    parameter "[COMMAND] ...", "Run command on host"
+
+    banner "Opens SSH sessions to hosts in the Kontena Pharos cluster."
+
+    option ['-r', '--role'], 'ROLE', 'select a server by role'
+    option ['-l', '--label'], 'LABEL=VALUE', 'select a server by label, can be specified multiple times', multivalued: true do |pair|
+      Hash[*[:key, :value].zip(pair.split('=', 2))]
+    end
+    option ['-a', '--address'], 'ADDRESS', 'select a server by public address'
+    option ['-p', '--private-address'], 'ADDRESS', 'select a server by private address'
```

Comment:
Not sure we need `--private-address`?

Reply:
It's only for filtering hosts, not  used for the ssh connection. You could see a private IP in some logs and use that to select the host to SSH into, but most likely nobody will know or remember that it's possible so maybe it's not useful.